src/test/e2e/cypress/e2e/search.cy.js

```diff
@@ -2,97 +2,99 @@ beforeEach(() => {
     cy.visit("/search");
 })
 
-it('should show info message on search page', () => {
-    cy.get('body').then((body) => {
-        expect(body).to.contain("Suche nach einer Prüfung...")
+describe('Verify content of search page', () => {
+    it('should show info message on search page', () => {
+        cy.get('body').then((body) => {
+            expect(body).to.contain("Suche nach einer Prüfung...")
+        })
     })
-})
-
-it('should display header on search page', () => {
-    cy.get('#logo').should("be.visible");
-})
 
-it('should display label for searchbar', () => {
-    cy.get('body').then((body) => {
-       cy.contains("Prüfungsnummer eingeben:")
+    it('should display header image on search page', () => {
+        cy.get('#logo').should("be.visible");
     })
-});
 
-it('should display input for exam number', () => {
-    cy.get('input[type="text"]').should('exist');
-});
-
-it('should display results when searching with valid exam number', () => {
-    cy.get('input[type="text"]').type('11000');
-    cy.get('button').click();
-    cy.contains('Privatrecht');
-});
+    it('should display label for searchbar', () => {
+        cy.get('body').then(() => {
+            cy.contains("Prüfungsnummer eingeben:")
+        })
+    });
 
-it('should display error when no exams were found', () => {
-    cy.get('input[type="text"]').type('99999');
-    cy.get('button').click();
-    cy.contains('Keine Prüfungen für die Prüfungslaufnummer 99999 gefunden');
-});
+    it('should display input for exam number', () => {
+        cy.get('[data-testid="exam-number-input"]').should('exist');
```

Comment:
Is there a reason why ` getByTestId()` is not used here? 
```suggestion
        cy.getByTestId("exam-number-input").should('exist');
```